dev-site.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+const fs = require('fs');
+const path = require('path');
+
+// Parse hash from app.1234abc.js
+// Ok to assume there's only one such file in /static
+function getStaticVersion() {
+  return fs.readdirSync(path.join(__dirname, 'static'))
+    .reduce((carry, filename) => {
+      const matches = /^app\.(\w+)\.js$/.exec(filename);
+      return matches ? matches[1] : carry;
+    }, null);
+}
+
+// Update app and vendor filenames with provided version or pick from /static
+const version = process.argv[2] || getStaticVersion();
+if (version) {
```

Comment:
The webpack config already has this setup with the HtmlWebpackPlugin. We can change the output directory to /static and then this logic becomes redundant.